### PR TITLE
Announce HHVM 4.159

### PR DESCRIPTION
**Pull request by Atry**

### CODE REVIEW

File: _posts/2022-04-26-hhvm-4.159.markdown [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: "HHVM 4.159"
+layout: post
+author: atry
+category: blog
+---
+
+HHVM 4.159 is released! This release marks the end of support of 4.153; 4.154 -
```
Comment (lexidor) [inline _posts/2022-04-26-hhvm-4.159.markdown:8]: 4.153 is an lts release. It is not losing support this week.

File: _posts/2022-04-26-hhvm-4.159.markdown [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+4.158 remain supported, as does the 4.128 LTS release.
+
+# Highlights
+
+- Add an attribute `<<__SafeGlobalVariable>>` to mark global variables as safe,
```
Comment (lexidor) [inline _posts/2022-04-26-hhvm-4.159.markdown:13]: This attribute has been renamed at HEAD. Might be worth adding a note here for future readers. https://github.com/facebook/hhvm/commit/eea2b351e1db8370b496c0f0108e9e7898ee0764

Comment (Atry) [reply _posts/2022-04-26-hhvm-4.159.markdown]: Thank you!

File: _posts/2022-04-26-hhvm-4.159.markdown (hunk 1 above)
Comment (Atry) [reply _posts/2022-04-26-hhvm-4.159.markdown]: Thanks for the reminder!

File: _posts/2022-04-26-hhvm-4.159.markdown [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+title: "HHVM 4.159"
+layout: post
+author: atry
+category: blog
+---
+
+HHVM 4.159 is released! 4.154 -
+4.158 remain supported, as does the 4.128 and 4.153 LTS release.
```
Comment (lexidor) [inline _posts/2022-04-26-hhvm-4.159.markdown:9]:
```suggestion
4.158 remain supported, as does the 4.128 and 4.153 LTS releases.
```

**Comment by AndrewDiMola** [comment]
Looks like there's two things here:
* Update https://docs.hhvm.com/hack/silencing-errors/error-codes#4005-array-access-on-a-type-that-doesnt-support-indexing from 4005 --> 4324.
* On https://docs.hhvm.com/hack/attributes/predefined-attributes, Add `__SafeGlobalVariable` 
*  (`__SafeForGlobalWriteCheck`); maybe just wait until it's renamed in the next release.

**Comment by AndrewDiMola** [comment]
Actually, I should probably add the 4.153 LTS here as well: https://docs.hhvm.com/hhvm/installation/release-schedule. Just noticed it's missing.

https://hhvm.com/blog/2022/03/17/hhvm-4.153.html

**Review by AndrewDiMola — APPROVED** [review]
Looks good for merge after @lexidor's new feedback.

**Comment by AndrewDiMola** [comment]
> > * On https://docs.hhvm.com/hack/attributes/predefined-attributes, Add `__SafeGlobalVariable`
> > * (`__SafeForGlobalWriteCheck`); maybe just wait until it's renamed in the next release.
> 
> Also attributes `__PolicyShardedMemoize` and `__PolicyShardedMemoizeLSB` are added to 4.159. I didn't mention it in this release note because it is used with `zoned`, which is considered as an internal feature, not for public use.

I can add them as internal notes.

**Pull request merged**